utils/dataset.py

```diff
@@ -10,6 +13,7 @@
 class Dataset(data.Dataset):
 
     def __init__(self, root='./', load_set='train', transform=None):
+        sys.stdout.write(f'{root}\n\n')
```

Comment:
Use `print()`
Don't use newlines like this.   

Make it more informational.   
`print("Dataset Root set to: {root}")`  or something.

Think about how many just raw variables you are printing to the console in for loops and how to differentiate them.